docs/excel/custom-functions-overview.md

```diff
@@ -28,11 +28,12 @@ See the Known Issues section at the end of this article, which includes current
 In the cloned sample repo, you’ll see the following files:
 
 -   customfunctions.js, which contains:
```

Comment:
@saunders77 - Please remove duplicate sentence. Thanks! 

I'll check the style guidance for filename styles.